x-pack/legacy/plugins/integrations_manager/server/integrations/handlers.ts

```diff
@@ -24,53 +34,39 @@ interface DeleteAssetRequest extends Request {
 }
 
 type AssetRequestParams = PackageRequest['params'] & {
-  asset: string;
+  asset?: string;
```

Comment:
It’s optional in [the actual pattern](https://github.com/jfsiii/kibana/blob/b4fbc22a6078269b44827e6b66a19817a06ff56b/x-pack/legacy/plugins/integrations_manager/common/routes.ts#L11-L12)